src/sys/mman.rs

```diff
@@ -424,12 +424,11 @@ pub unsafe fn mmap(
     fd: RawFd,
     offset: off_t,
 ) -> Result<*mut c_void> {
-    let ptr = addr.map_or(
-        std::ptr::null_mut(),
-        |a| usize::from(a) as *mut c_void
-    );
-    
-    let ret = libc::mmap(ptr, length.into(), prot.bits(), flags.bits(), fd, offset);
+    let ptr =
```

Comment:
There's more formatting changes like this mixed in with content changes.  Can you please move them into separate commits?

Reply:
I thought this was due to the edition bump, but it looks like this is because I'm on nightly? When I run a cargo fmt on master this stuff changes. Created https://github.com/nix-rust/nix/pull/1909